lib/appraisal/gemfile.rb

```diff
@@ -1,26 +1,14 @@
-require 'appraisal/dependency_list'
-require 'appraisal/gemspec'
-require 'appraisal/git_source'
-require 'appraisal/path_source'
-require 'appraisal/group'
-require 'appraisal/platform'
+require 'appraisal/bundler_dsl'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.